test/client/fixtures/sandbox/xhr-test.js

```diff
@@ -69,10 +73,23 @@ test('createNativeXHR', function () {
     }
 });
 
+test('constructor', function () {
```

Comment:
`toString, instanceof, constructor and static properties`
or simple `properties`